client/components/forms/docs/example.jsx

```diff
@@ -35,7 +36,10 @@ import PhoneInput from 'components/phone-input';
 /**
  * Internal dependencies
  */
-var countriesList = require( 'lib/countries-list' ).forSms();
+const countriesList = require( 'lib/countries-list' ).forSms();
+const currencies = require( 'lib/format-currency/currencies' ).CURRENCIES
```

Comment:
Are the `lib/countries-list` and `lib/format-currency/currencies` modules really being loaded async here? I don't see the mechanism how this would work -- some Babel or Webpack magic?

I thought that `require` is used here only to conveniently call `forSms()` and other methods directly on the return value of `require()`.

It's a good idea to load these long lists of countries and currencies async instead of bundling them statically into the compiled chunks. We should do that in a follow-up PR if it's not happening already.